rotkehlchen/accounting/transactions.py

```diff
@@ -25,15 +26,15 @@ def evm_events_iterator(
     Takes associated event as an argument to be able to log it in case of errors.
     """
     for event in events_iterator:
-        if not isinstance(event, EvmEvent):
+        if not isinstance(event, EvmEvent) and not isinstance(event, HistoryEvent):  # TODO: undo. Is a temporary hack  # noqa: E501
```

Comment:
> but since HistoryBaseEntry > EvmEvent isn't it enough to do (unless you explecitly want to restrict the type and avoid the eth staking events):

Yeah, I used HistoryEvent + EvmEvent to be more explicit